_includes/tutorials/connect-sink-timestamp/ksql/markup/dev/make-docker-compose.adoc

```diff
@@ -0,0 +1,5 @@
+Next, create the following `docker-compose.yml` file to obtain Confluent Platform:
```

Comment:
This docker-compose doesn't just obtain Confluent Platform, it also spins up the postgres database. I think it would be helpful to either make postgres bringup a discrete step, or expand the text to tell the reader the full extent of this docker compose.

Reply:
That is a good point, a lot of "stuff" happens in this docker-compose. Similar to a comment above, I did address this at a high level. (https://github.com/confluentinc/kafka-tutorials/pull/760#discussion_r595541683) I can go into more detail if you'd like.

Reply:
LGMT